src/schemas/transfer-benefits/schema.js

```diff
@@ -8,7 +8,7 @@ let schema = {
   type: 'object',
   additionalProperties: false,
   definitions: _.pick(definitions,
-    'privacyAgreementAccepted',
+    'privacyAgreementAccepted', [
```

Comment:
You're right, I don't think they're necessary, I was just copying from change-of-program, which has them. I can remove.